containers/codespaces-linux/.devcontainer/Dockerfile

```diff
@@ -26,9 +26,9 @@ ENV SHELL=/bin/bash
 ENV ORYX_ENV_TYPE=vsonline-present
```

Comment:
We can change this too, but let Oryx team know.

Reply:
Yep thats why I haven't changed it yet.